applications/suit/flash_companion/README.rst

```diff
@@ -0,0 +1,84 @@
+.. _suit_flash_companion:
+
+SUIT flash companion
```

Comment:
Has the name of the application been checked with the respective product owners? This is usually the procedure for the applications that has been followed for apps like nRF Machine Learning, nRF Desktop, both weather stations etc.

If it has been checked, is the capitalization correct? The application names are usually capitalized.

Reply:
@wbober

Reply:
I have talked with @wbober  and I'll be moving this application to samples directory. I'll make a push later today.

Reply:
Okay, thanks Rafal. 
Will review once that is done and other comments have been addressed.

Please also let me know/we can discuss as a team with Maciej (does not need to be done in _this_ PR) if and where this sample should be mentioned/linked to in other SUIT docs.

Reply:
The application was moved to `samples/suit` directory.